app/src/main/java/org/openobservatory/ooniprobe/activity/ResultDetailActivity.java

```diff
@@ -71,20 +80,50 @@ public static Intent newIntent(Context context, int id) {
 		LinearLayoutManager layoutManager = new LinearLayoutManager(this);
 		recycler.setLayoutManager(layoutManager);
 		recycler.addItemDecoration(new DividerItemDecoration(this, layoutManager.getOrientation()));
-		ArrayList<HeterogeneousRecyclerItem> items = new ArrayList<>();
+		result.is_viewed = true;
+		result.save();
+		items = new ArrayList<>();
+		adapter = new HeterogeneousRecyclerAdapter<>(this, items);
+		recycler.setAdapter(adapter);
+		snackbar = Snackbar.make(coordinatorLayout, R.string.Snackbar_ResultsSomeNotUploaded_Text, Snackbar.LENGTH_INDEFINITE).setAction(R.string.Snackbar_ResultsSomeNotUploaded_UploadAll, v1 -> new MKCollectorResubmitSettingsAsyncTask(this).execute(result.id, null));
```

Comment:
There is a lot of stuff going on in this line.

Maybe the inline function definition could deserve it's own line and a comment?

Reply:
wrapped

Reply:
You did not move the inline function to it's own line.

Reply:
done